demonstrations/tutorial_vqe_uccsd_obs.py

```diff
@@ -0,0 +1,483 @@
+r"""
+VQE in different spin sectors
+=============================
+
+.. meta::
+    :property="og:description": Find the lowest-energy states of a Hamiltonian in different
+        spin sectors using the variational quantum eigensolver
+        algorithm in PennyLane.
```

Comment:
"Using the variational quantum eigensolver algorithm in PennyLane" sounds a bit un-needed? The metadata should be short and snappy where possible

```suggestion
    :property="og:description": Find the lowest-energy states of a Hamiltonian in different spin sectors
```